setup.py

```diff
@@ -94,6 +94,9 @@ def run(self):
             'env-docker/*',
             'retry-docker/*'
         ],
+        # These data files are used by the volumes API to define input and
```

Comment:
Yes, it seemed to be. 
Adding the files to MANIFEST ensures that they are included in an sdist package.
Adding to `package_data` ensures that those files are actually installed.